src/passes/literalExpressionEvaluator/literalExpressionEvaluator.ts

```diff
@@ -23,10 +24,14 @@ import { RationalLiteral, stringToLiteralValue } from './rationalLiteral';
   >, <, <=, >=
   ==, !=
   &&, ||
-
   Hence all literal nodes are of type Literal | UnaryOperation | BinaryOperation
 */
 
+interface NodeEvaluationInfo {
+  evaluation: RationalLiteral | boolean | null;
+  node: ASTNode;
+}
+
```

Comment:
This is just a nitpick but I've noticed through out the code that we prefer `type NodeEvaluateInfo = {...}` to `interface ...`.

Reply:
True